test-tools/wamr-ide/WASM-Toolchain/Docker/Dockerfile

```diff
@@ -31,18 +28,16 @@ RUN wget --progress=dot:giga https://github.com/WebAssembly/wasi-sdk/releases/do
     && rm -f wasi-sdk-*-linux.tar.gz
 
 ##  - clone wamr repo
-RUN git clone -b main --depth=1 https://github.com/bytecodealliance/wasm-micro-runtime.git
+RUN git clone -b main --depth=1 https://github.com/bytecodealliance/wasm-micro-runtime.git \
+    && cd wasm-micro-runtime/build-scripts \
+    && pip3 install --user -r requirements.txt
```

Comment:
just my little concern, does it work well without installing *pip*, *ninja* and *CCache*?

Reply:
got it